### DISCUSSION TITLE
Vercel部署报错出现这种状况改如何解决

### DISCUSSION THREAD

**Discussion by LYK1013** [lobehub/lobehub#1443] [Self Hosting]
不管是一键部署还是fork后在Vercel选择仓库部署，都会出现这种报错，这种情况该如何解决。
![Snipaste_2024-03-03_03-15-58](https://github.com/lobehub/lobe-chat/assets/44458329/585c4cd0-06a7-4bf0-815d-b2cb34195bb3)
![Snipaste_2024-03-03_03-16-09](https://github.com/lobehub/lobe-chat/assets/44458329/cc3c6bed-bebf-4217-a683-47da5b97438b)
![Snipaste_2024-03-03_03-16-17](https://github.com/lobehub/lobe-chat/assets/44458329/1752b183-41c0-42ec-9259-e79837ad69f9)

**Comment by arvinxx** [comment]
已修复，请重试下

**Reply by LYK1013** [reply]
感谢🙏
